mobile/src/full/java/org/openhab/habdroid/core/FcmRegistrationService.kt

```diff
@@ -66,6 +67,9 @@ class FcmRegistrationService : JobIntentService() {
                 } catch (e: HttpClient.HttpException) {
```

Comment:
Can Kotlin catch multiple exceptions in one `catch` block?

Reply:
Unfortunately not (yet): https://youtrack.jetbrains.com/issue/KT-7128